src/Compilers/CSharp/Test/Syntax/Parsing/UsingDirectiveParsingTests.cs

```diff
@@ -5257,4 +5258,143 @@ public void UsingStatic_UnsafeType_CSharp12_UnsafeFlag()
         }
         EOF();
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/67989")]
+    public void AliasToPointerReferencedInExpression()
```

Comment:
I do not think this test belongs in parsing tests. There is nothing unique about this scenario from the parsing perspective. The test should be moved to one of the binding related assemblies. For example, we have a ```UsingAliasTests``` type in Symbol tests.

Reply:
I was previously asked to try to keep all the tests for a particular feature in single location.  Maybe we can change this from UsingDirectiveParsingTests, to just UsingDirectiveTests?

Reply:
> I was previously asked to try to keep all the tests for a particular feature in single location. 

Could you provide a link? Perhaps there was some confusion, we do not mix parsing tree tests with semantic tests, and. I believe, that started happening with changes that moved error reporting between layers in order to keep verifying diagnostics in the original tests. An exception like this is not applicable to new tests, especially those that have nothing to do with anything parsing related.